bodhi/server/consumers/masher.py

```diff
@@ -721,6 +735,37 @@ def sanity_check_repo(self):
 
         return True
 
+    def wait_for_repo_signature(self):
+        """Wait for a repo signature to appear if configured as such."""
+        if not config.get('wait_for_repo_sig'):
+            self.log.info('Not waiting for a repo signature')
+            return
```

Comment:
Optional suggestion: Why not move this if statement to the place where ```wait_for_repo_signature()``` is called? That way, this method always waits if called, which is more like what its name sounds like.

If you like this better I'm not opposed, just a suggestion.

Reply:
I primarily went with it here because the place where it gets called is really a single series of statements, with just the "if mashing;" conditionals. I'll move it over.